SPECS/byaccj/byaccj.spec

```diff
@@ -1,31 +1,13 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-#
-# spec file for package byaccj
-#
-# Copyright (c) 2014 SUSE LINUX Products GmbH, Nuernberg, Germany.
```

Comment:
Same comment for the copyright notice here and for any other spec, which might need it restored.